src/components/crossDevice/Intro/style.css

```diff
@@ -0,0 +1,55 @@
+.content {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  justify-content: center;
+}
+
+.step {
+  display: flex;
+  padding-bottom: 32px;
+  position: relative;
+
+  &::before {
```

Comment:
what will this style?

Reply:
This is a line that connects all three icons:

![image](https://user-images.githubusercontent.com/1504258/42461661-698e96aa-8377-11e8-8f85-550a3e76c373.png)

it needs to stretch from top to bottom, and adjust if copy splits into more/less lines.

Reply:
I see. Might be good to add that as comment otherwise it will be hard to find out from the code.